molscrub/utils.py

```diff
@@ -8,29 +8,44 @@
 
 import numpy as np
 
+from .espaloma_minim import EspalomaMinimizer
+
 def optimize_conformers(mol: Mol, ff: str="mmff94", max_ff_iter: int = 400):
```

Comment:
Could you add a `mol = Chem.Mol(mol)` to make a copy of the molecule at the beggining of the function? Then MMFF and UFF will **not** modify the state (conformers) of the input mol. Currently, espaloma does not modify the input mol, but UFF/MMFF do, and returning a mol hints (at least to me) that the input is not modified (otherwise there's no reason to return a new mol).